.gitlab-ci.yml

```diff
@@ -71,7 +71,7 @@ ci-not-authorized:
   after_script: []
   rules:
     # LGTM or ok-to-test labels
-    - if: $PR_LABELS =~ /.*,(lgtm|ok-to-test).*|^(lgtm|ok-to-test).*/i
+    - if: $PR_LABELS =~ /.*,(lgtm|approved|ok-to-test).*|^(lgtm|approved|ok-to-test).*/i
```

Comment:
no sure about this change. 
approvers can also add the /ok-to-test flag themself.  
idea is to no automatically trigger pipeline for PR that are not ready to be tested (including for reviewer/approvers)